src/app/rate/Form.js

```diff
@@ -35,7 +35,9 @@ export default function Form({ name = "", owner = "" }) {
 
     if (data.success) {
       setSending(false);
-      redirect("/?alert=success&message=Rating saved!");
+      redirect(
+        `/?alert=success&message=Rating%20saved!%20for%20${owner}/${name}%20🎉🎉`
```

Comment:
I don't think it needs url encoding? It makes it more difficult to read in the code - please test without encoding

Reply:
> I don't think it needs url encoding? It makes it more difficult to read in the code - please test without encoding


Removed URL encoding and it is still working.

Reply:
> > I don't think it needs url encoding? It makes it more difficult to read in the code - please test without encoding
> 

Removed URL encoding and it is still working.

Screenshot:(works for long messages too)
![Screenshot 2024-01-07 235955](https://github.com/EddieHubCommunity/RepoRater/assets/124811276/501c4152-6e52-41ba-bcfb-d2216f08f136)